lib/models/query_metadata.js

```diff
@@ -19,49 +19,47 @@ class QueryMetadata {
      * Returns the calculated X-Cache-Channel for all of the tables.
      * @returns {String}
      */
-    getCacheChannel(skipNotUpdatedAtTables) {
+    getCacheChannel (skipNotUpdatedAtTables) {
         const groupedTables = this.getTables(skipNotUpdatedAtTables).reduce((grouped, table) => {
-            if (!grouped.hasOwnProperty(table.dbname)) {
+            if (!Object.prototype.hasOwnProperty.call(grouped, table.dbname)) {
```

Comment:
What's with these changes (a.hasOwnProperty -> Object.prototype...)? I don't think it's more readable now than before.

Reply:
https://eslint.org/docs/rules/no-prototype-builtins

Reply:
Worse readability but safer.... :(

Reply:
:ok_hand: